pallets/minterest-protocol/src/lib.rs

```diff
@@ -466,6 +483,9 @@ impl<T: Config> Pallet<T> {
 
 		<Controller<T>>::accrue_interest_rate(underlying_asset).map_err(|_| Error::<T>::AccrueInterestFailed)?;
 
+		<T as module::Config>::MntManager::update_mnt_supply_index(underlying_asset)?;
+		<T as module::Config>::MntManager::distribute_supplier_mnt(underlying_asset, who, false)?;
```

Comment:
```suggestion
		T::MntManager::update_mnt_supply_index(underlying_asset)?;
		T::MntManager::distribute_supplier_mnt(underlying_asset, who, false)?;
```
And the same for other places

Reply:
Fixed